### ISSUE TITLE
Move "Introduction to Cloud Computing for Genomics" to the beginning of workshop as most students struggle to get onto the AWS instance?

### ISSUE THREAD

**Issue by waydeveldman** [datacarpentry/genomics-workshop#124]
Before the workshop even begins, most students struggle to get onto the AWS instance. Perhaps we should start the workshop with "Introduction to Cloud Computing for Genomics", or at least the "Logging onto a cloud instance" part?

**Comment by bio2data** [comment]
I agree, this was my first time teaching this workshop and I thought that having the "intro to cloud computing" at the ends was a not intuitive. I was in charge of teaching the data wrangling portion. Before beginning I had everyone access the remote server. I also showed a slide to explain what the cloud was (a very brief explanation). This gave me time to help the students get connected to the AWS before the coding portion began. And by explaining what a remote server was, they were better able to grasp the concept of an "AWS" connection. We then told the students we would dive deeper into cloud computing at the end of the workshop. 

here were my slides if anyone is interested (specifically page 4-6). 
[BriefNotes.pptx](https://github.com/datacarpentry/genomics-workshop/files/8491919/BriefNotes.pptx)

**Comment by kerchner** [comment]
I concur and will submit a PR to reorder this, and the reviewers can make the decision.   Having taught the lesson several times, I always move the Intro to Cloud Computing to precede the Command Line lesson.

I also noticed that the "Common Schedules A/B/C" section omits Intro to Cloud Computing.  Not sure why that is; I'll add it in, noting that it does add around an hour to the content.
